modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc

```diff
@@ -225,13 +258,13 @@ bool IterativeAnchoringSmoother::SmoothPath(
 
   FemPosDeviationSmoother fem_pos_smoother;
   // TODO(Jinyun): move to confs
-  fem_pos_smoother.set_weight_fem_pos_deviation(1e5);
+  fem_pos_smoother.set_weight_fem_pos_deviation(1.0e8);
```

Comment:
Better reformulate the problem for dimension normalization in next iteration, to increase robustness and make future tuning easier.